tensorflow/contrib/makefile/download_dependencies.sh

```diff
@@ -19,7 +19,9 @@ set -e
 DOWNLOADS_DIR=tensorflow/contrib/makefile/downloads
 BZL_FILE_PATH=tensorflow/workspace.bzl
 
-EIGEN_URL="$(grep -o 'http.*bitbucket.org/eigen/eigen/get/.*tar\.gz' "${BZL_FILE_PATH}" | grep -v bazel-mirror | head -n1)"
+# Temporary fix to refer to Benoit Steiner's Eigen repository instead of mainline.
+#EIGEN_URL="$(grep -o 'http.*bitbucket.org/eigen/eigen/get/.*tar\.gz' "${BZL_FILE_PATH}" | grep -v bazel-mirror | head -n1)"
+EIGEN_URL="https://bitbucket.org/benoitsteiner/opencl/get/123510871143.tar.gz"
```

Comment:
Does it have to be fixed to this particular revision?
Couldn't we do: 
EIGEN_URL="$(grep -o 'http.*bitbucket.org/benoitsteiner/eigen/get/.*tar\.gz' "${BZL_FILE_PATH}" | grep -v bazel-mirror | head -n1)"